hyperspy/_signals/lazy.py

```diff
@@ -258,7 +258,7 @@ def __array__(self, dtype=None):
     def _unfold(self, *args):
         raise lazyerror
 
-    def _make_sure_data_is_contiguous(self, log=None):
+    def _make_sure_data_is_contiguous(self):
```

Comment:
Even though ``log`` is not used here, we must put it back as it is used in the non-lazy function and call to the method may use the arg now or in the future.

Reply:
I don't see where it is used in his non-lazy counterpart, the logger is used and not the log argument. Could it be a legacy from the time, when there was no logger? Or am I missing something?

Reply:
Here https://github.com/hyperspy/hyperspy/blob/7932b6c44fea40991883a97a9a42a5ae301b8b1b/hyperspy/signal.py#L2690

Reply:
Yes, sorry I was looking in the wrong branch... reverted it back now!